File: Problem_Set_6/sentimental-mario-more/mario.py
from cs50 import get_int

def main():
    height = 9
    while height < 1 or height > 8:
        height = get_int("Height: ")

    if height == 1:
        print("#  #")
    else:
        print_case(height)

def print_case(n):
    j = 1
    while j < n + 1:
        for k in range(0, n - j):
            print(" ", end="")
        for l in range(j, 0, -1):
            print("#", end="")
        print("  ", end="")
        for m in range(0, j):
            print('#', end="")
        print()
        j += 1

if __name__ == "__main__":
    main()

File: Problem_Set_1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

/* function declaration*/
int prompt(void);
void printStair(int n);
/* variable declaration*/
int height;

int main(void)
{
 height = prompt();
 if ( height==1 ){
    printf("# #\n");
 }
 else
 {
    printStair(height);
 }
}

int prompt(void)
{
    do
  {
      height = get_int("Enter height size: ");
  }
  while (height < 0);
  return height;
}

void printStair(int n){
    int j = 1;
    while (j < n + 1) {
        for (int k = 0; k < n-j ; k++ ) {
            printf(" ");
        }
        for (int l = j; l > 0; l--) {
            printf("#");
        }

        printf(" ");

        for (int m = 0; m < j ; m++ ) {
            printf("#");
        }
        printf("\n");
        j = j + 1;
    }
}